tests/functional/spawn-on-demand/07-abs-triggers/suite.rc

```diff
@@ -13,9 +13,9 @@
 [runtime]
     [[start]]
         script = """
-# Ensure that bar.1,2 are spawned by foo.1,2 not by start.2 (so the scheduler
-# must update their prereqs in the pool after start.2 is done).
-cylc__job__poll_grep_suite_log "bar\.2.*spawned"
+# Ensure that bar.1,2 are spawned by foo.1,2 and not by start.2
+# (so the scheduler must update their prereqs when start.2 finishes).
+cylc__job__poll_grep_suite_log "spawned bar\.2"
```

Comment:
These task spawning log messages are of the form:
```
(foo.2) spawned bar.2
```
The original line in this test was looking for `bar.2` spawning another task; it should have been looking for `bar.2` being spawned (which could either be by `foo.2` or `bar.1` depending on timing, in this test).

Reply:
Thanks! Read the old comments and code, followed by your explanation above. Makes sense. Just waiting for CI to finish to merge :+1: